src/components/presentational/TimeSeriesChart/TimeSeriesChart.tsx

```diff
@@ -228,12 +284,16 @@ export default function TimeSeriesChart(props: TimeSeriesChartProps) {
             }
             {props.chartHasData && props.chartType === "Line" &&
                 <ResponsiveContainer width="100%" height={150}>
-                    <LineChart width={400} height={400} data={dataToDisplay} syncId="DailyDataChart">
+                    <LineChart width={400} height={400} data={dataToDisplay} syncId="DailyDataChart" margin={{ left: 5, top: 5, bottom: 5, right: 40 }}>
```

Comment:
I think these margins were just removed in this PR:  https://github.com/CareEvolution/MyDataHelpsUI/pull/268/files#r1644688975